ports/silabs/README.md

```diff
@@ -115,18 +114,18 @@ At the boot stage, two scripts will be run (if not booting in safe mode). First,
 
 After code.py has finished executing, a REPL prompt will be presented on the serial port. Other files can also be executed by using the **Thonny** editors or using **Ampy** tool.
 
-![Thony](./res/Thony.png)
+![Thonny](./res/Thonny.png)
 
 With the boards which support USB mass storage, we can drag the files to the board file system. However, because the EFR32 boards don’t support USB mass storage, we need to use a tool like **Ampy** to copy the file to the board. You can use the latest version of **Ampy** and its  command to copy the module directories to the board.
```

Comment:
I saw the use of Ampy here, it's currently maintained by @scientifichackers but the attached learn guide seems outdated. Not sure if there's anything we can do here.

Reply:
Generally we suggest Thonny now.